frontend/components/feedme/FolderTreeViewSimple.tsx

```
@@ -9,32 +9,49 @@
 import { ScrollArea } from '@/components/ui/scroll-area'
+import { Dialog, DialogContent, DialogHeader, DialogTitle, DialogFooter } from '@/components/ui/dialog'
+import { Input } from '@/components/ui/input'
+import { Label } from '@/components/ui/label'
 
 interface FolderTreeViewProps {
   onConversationSelect?: (conversationId: number) => void
   expanded?: boolean
   className?: string
 }
 
+interface Folder {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Folder'. Consider to delete it or rename it.

'Folder' is defined here:


(lint/suspicious/noRedeclare)